frontend/src/components/Profile/Profile.tsx

```diff
@@ -1,7 +1,55 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { AccountCircle } from '@mui/icons-material';
+import Button from '@mui/material/Button';
 
-export const Profile: React.FC = () => (
-  <>
-    <h1>This is the profile component</h1>
-  </>
-);
+export const Profile: React.FC = () => {
+
+    const [formData, setFormData] = useState({
+        name: '',
+        email: '',
+        phone: '',
+        address: '',
+        company: '',
+        description: ''
+    });
+
+    const handleEditClick = () => {
+        console.log("Edit button clicked!");
+    };
+
+    const handleInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+        const { name, value } = e.target;
+        setFormData(prev => ({ ...prev, [name]: value }));
+        console.log("Information edited");
+    };
+
+    return (
+        <div className="absolute inset-0 h-screen w-screen flex items-center justify-center bg-gray-100">
+            <div className="bg-white p-8 rounded-lg shadow-md w-4/5 h-4/5">
+                <div className="animate-pulse flex h-full">
+                    <div className="relative w-1/4 h-1/4 flex flex-col items-center">
+                        <div className="w-2/5 h-4/5 rounded-full bg-gray-300 flex items-center justify-center">
+                            <AccountCircle style={{ fontSize: '100%', position: 'absolute', top: '40%', left: '50%', transform: 'translate(-50%, -50%)', width: '80%', height: '80%' }}/>
+                        </div>
+                        <Button 
+                            style={{top:"10%", width:"35%"}}
+                            variant="contained" 
+                            color="primary" 
+                            className="w-3/5 mt-2"
+                            onClick={handleEditClick}>
+                            Edit
+                        </Button>
+                    </div>
+                    <div className="ml-6 flex flex-col justify-between w-3/4 space-y-4">
+                        <input name="name" value={formData.name} onChange={handleInputChange} className="h-20 bg-gray-300 rounded w-3/4 text-xl text-black pl-4" placeholder="Name" />
```

Comment:
If you're interested, Material UI does have a TextField component. It could mean less code for us to write. Thoughts @KevinLu9?
https://mui.com/material-ui/react-text-field/

Reply:
I think this should be better to use the text component from MUI, I did not realise I can use that.
Here is the screenshot of current profile page.


![profile_1](https://github.com/mursal-ashraf/bankan/assets/100513480/d5fe0d7b-7b53-4214-82f6-67478f5cce6e)

Reply:
Thanks Dongyi! If MUI components can be added, then PR is good to go! Page looks 👌

Reply:
Sure! I am going to add the MUI part and try to make the UI looks better. Gonna finish them before the Sunday

Reply:
Yea, i think we should try to use MUI whenever possible.